content/altinn-studio/reference/ux/pages/summary/_index.en.md

```diff
@@ -9,7 +9,9 @@ weight: 30
 {{%notice warning%}}
 Setup must be done manually as of today. Support for setup through Altinn Studio will be available at a later point in time.
 
-**NOTE**: The PDF generation, as of today, does not support the summary component. For PDF generating to work, either all the summary components or the whole summary-page(s) must be excluded from PDF. This is done in the `Settings.json`-file attached to the layout-files. 
+<br>
+
+**NOTE**: The PDF generation does not show any summary components. For PDF generation to work properly, you may want to exclude some components or whole summary-pages from the PDF. This is done in the `Settings.json` file associated with the layout-files, see [Excluding pages and components](/altinn-studio/reference/ux/pdf/#excluding-pages-and-components). 
```

Comment:
Slightly confusing? I get it after reading this the third time (it's saying that we ignore plain summary components when auto-generating a PDF layout? But of course, we do generate new summary components for existing form components).

To me it seems like the extra notice about the 'exclude pages and components' functionality is something else entirely? If we already do not show summary components, why would you want to exclude them?

Reply:
Hmm, should I just delete the note on PDF? I don't think it belongs there anyway 🤔

Reply:
Hmm, yep, that may be better than the confusion. :raised_hands:

Reply:
Flytta den merknaden til siden om PDF under automatisk PDF